api/index.ts

```diff
@@ -117,18 +117,44 @@ const v1Routes = (cfg: APIConfig, store: Store) => async (server: FastifyTypebox
   }
 
   // Get global list of blocked users/instances as newline delimited string
-  server.get('/blocklist', async (request, reply) => {})
+  server.get('/blocklist', async (request, reply) => {
+    const blockedAccounts = await store.blocklist.list();
+    reply.send(blockedAccounts.join('\n'));
```

Comment:
We should set th3e content type to text/plain or similar if it isn't already

Reply:
Also could you add types to the route the way I have them in the inbox? (same ofr the other endpoints)